pywbemtools/pywbemlistener/_cmd_listener.py

```diff
@@ -278,6 +294,11 @@ def created(self):
 
 @cli.command('run', cls=PywbemtoolsCommand, options_metavar=CMD_OPTS_TXT)
 @click.argument('name', type=str, metavar='NAME', required=False)
+@click.option('--start-pid', type=str, metavar='PID',
```

Comment:
This one may require more explanation here or in doc as to why it is required.

Reply:
DONE
Added this sentence:
"Note: The --start-pid option is needed because on Windows, the
    `pywbemlistener run` command is not the direct child process of the
    `pywbemlistener start` command starting it."